Fix minor grammatical and spelling errors (#30407)
freeCodeCamp/freeCodeCamp: guide/english/cplusplus/if-else-statement/index.md

### Another example of if...else if...else ladder

> Suppose we have the user input two numbers and we are going to display if either number is greater than the other. And if neither is greater than the other, then we print the statement "Both are equal".

Reviewer: Suppose we have the user input two numbers to display
```suggestion
Suppose we ask the user to input two numbers then display greater than or equal. ```

Author: The suggested sentence is much more succinct, I am just unsure as to why with the additional back-ticks which were added at the end?
